src/Sarif/Readers/JsonPositionedTextReader.cs

```diff
@@ -37,6 +37,19 @@ public class JsonPositionedTextReader : JsonTextReader
         public JsonPositionedTextReader(string filePath) : this(() => File.OpenRead(filePath))
         { }
 
+
+        /// <summary>
+        /// Create a JsonPositionedTextReader based on a Stream and passes this to deferred collections.
+        /// </summary>
+        /// <param name="stream"></param>
+        public JsonPositionedTextReader(Stream stream) : this(() => new DelegatingStream(stream))
+        {
+            if (!stream.CanSeek)
```

Comment:
>stream [](http://example.com/codeflow?start=17&length=6)

too late! you already crashed in the DelegatingStream ctor when you tried to seek to position 0. right?

Reply:
we removed the seek to position 0.